app/views/records/edit_fields/_document_type_uris.html.erb

```diff
@@ -1,3 +1,8 @@
-<%= f.input key, as: :multi_value_select, collection: DocumentType.options, include_blank: true,
-    selected: f.object.uris_for(:document_type),
-    input_html: { class: 'form-control', multiple: true }, required: f.object.required?(key) %>
+<div class="form-group multi_value required generic_work_document_type_uris managed">
+  	<label class="multi_value required control-label" for="generic_work_document_type_uris">Document type <span class="label label-info required-tag">required</span></label>    
+	
+	<select class="select required form-control form-control" id="docTypes">
```

Comment:
id should be `generic_work_document_type_uris` and there also needs to be `name=generic_work[document_type_uris][]` so that the values will be passed in the parameters hash.